benchmarks/bench_minhash.py

```diff
@@ -0,0 +1,460 @@
+#################
+# The MinHashEncoder version used for the benchmark
+# On the main branch, we only kept the best version of the MinHashEncoder
+# which is the batched version
+# with batch_per_job=1
+# (the batch_per_job parameter has no effect on the results)
```

Comment:
From what I see, batches are useful in the end.
The detailed benchmarking results also depend on the number of batches per job, is it because the gain is not that big that we don't leave it as a choice?

Reply:
Yes the impact of `batch_per_job` is very small, and the impact of `batched` is large but seems quite constant. In both cases, I don't think it is worth it to have additional parameters, both for ease of use and ease of maintenance.